t/ingredients_nesting.t

```diff
@@ -309,7 +310,7 @@ foreach my $test_ref (@tests) {
 		# inside the test file much easier when tests results need
 		# to be updated. Caveat is that it might interfere with
 		# test output.
-		or print STDERR join("\n", explain $product_ref->{ingredients});
+		or diag join("\n", explain $product_ref->{ingredients});
```

Comment:
Will we still get this in the test output when the tests fail? We do need it.

Reply:
Yes, but better. 😄 `diag` just prints the message, but ensures that the order of messages in the test log is correct.